### PR TITLE
added a social sticky bar in home page

### PR DESCRIPTION
**Pull request by praneetha3108**

### CODE REVIEW

File: src/components/Homes/Carousel.vue [hunk 1]
```diff
@@ -1,5 +1,13 @@
 <template>
+
   <div>
+    <div class="icon-bar">
+ <a href="https://www.facebook.com/iiest.shibpur.official" class="facebook"><i class="fab fa-facebook-f"></i></a> 
+  <a href="https://twitter.com/iiest_s" class="twitter"><i class="fab fa-twitter"></i></a> 
+  <a href="https://iiests.irins.org/" class="vidwan"><i class="fas fa-lightbulb"></i></a> 
+  <a href="https://www.linkedin.com/school/iiests/" class="linkedin"><i class="fab fa-linkedin-in"></i></a>
+  <a href="shivamhere247@gmail.com" class="gmail"><i class="  fas fa-envelope"></i></a>
```
Comment (shivam-51) [inline src/components/Homes/Carousel.vue]: make it `href="mailto:shivamhere247@gmail.com"`

Comment (shivam-51) [reply src/components/Homes/Carousel.vue]: @praneetha3108  Nice idea! Should we remove the top-right hamburger now? If yes, add it in this patch itself

Comment (praneetha3108) [reply src/components/Homes/Carousel.vue]: > @praneetha3108 Nice idea! Should we remove the top-right hamburger now? If yes, add it in this patch itself

yeah I will add those hamburger kinks in this

Comment (shivam-51) [reply src/components/Homes/Carousel.vue]: @praneetha3108 Should we merge this patch and you can remove those other links in a separate patch, perhaps?

**Comment by praneetha3108** [comment]
Merger that I will create a new pull request

**Comment by shivam-51** [comment]
> Merger that I will create a new pull request
> […](#)

Fix the merge conflicts first. After that I can merge it

**Comment by shivam-51** [comment]
Praneetha you haven't resolved the conflicts yet.
Anyways I will do it :)

**Comment by praneetha3108** [comment]
No Don't merge

**Pull request merged**